.flowconfig

```diff
@@ -34,3 +34,7 @@ suppress_comment=\\(.\\|\n\\)*\\$FlowFixedInNextDeploy
 suppress_comment=\\(.\\|\n\\)*\\$FlowExpectedError
 
 [strict]
+
+
+[version]
+^0.89.0
```

Comment:
Bummer - looks like this is specified here and in pakcage.json. Why does it need to be set here?

Reply:
Flow could be installed locally or globally. It enforces the version being used to type the project regardless of where it is installed.

Reply:
Goctha - thanks!